packages/php/email-editor/src/Integrations/Core/Renderer/Blocks/class-cover.php

```
@@ -0,0 +1,226 @@
+		return $sanitized_color;
+	}
+
+	/**
+	 * Build the cover content with background image or color.
+	 *
+	 * @param string $inner_content Inner content.
+	 * @param string $background_image Background image URL.
+	 * @param string $background_color Background color.
+	 * @return string Cover content HTML.
+	 */
+	private function build_cover_content( string $inner_content, string $background_image, string $background_color ): string {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$background_image'. (undefined)

(UnusedFormalParameter)